apps/opencs/view/render/lightingbright.cpp

```diff
@@ -4,7 +4,7 @@
 
 CSVRender::LightingBright::LightingBright() {}
 
-void CSVRender::LightingBright::activate (osg::Group* rootNode)
+void CSVRender::LightingBright::activate (osg::Group* rootNode, bool isExterior)
```

Comment:
Perhaps "isExterior" should be commented to clarify that it's not going to be used. (bool /*isExterior*/).
The whole implementation of updateDayNightMode is pretty clunky with all these arguments being forwarded to each lighting mode, but I don't know what would be better.